profiles/NIST_SP-800-53_rev4_MODERATE-baseline_profile.json

```diff
@@ -2566,7 +2566,7 @@
           "description": "NIST Special Publication 800-53 Revision 4: Security and Privacy Controls for Federal Information Systems and Organizations",
           "rlinks": [
             {
-              "href": "https://raw.githubusercontent.com/usnistgov/oscal-content/main/nist.gov/SP800-53/rev4/json/NIST_SP-800-53_rev4_catalog.xml",
```

Comment:
To be honest when I saw this difference in the profile I was skeptical about his change as well. I tested it with the `OSCAL Viewer` and didn't see noticeable changes. IT seems that I should have check for the changes on the `oscalviewer`. On second thought I think this is one of the instances where it shouldn't exactly match NIST's content. It also wouldn't make sense to have an `href` that links to a file instead of a path (link). I've changed it back to the github reference